### ISSUE TITLE
Getting error when running container

### ISSUE THREAD

**Issue by creativeguitar** [GuitarML/PedalNetRT#30]
Thank you for working on this project!!!
is it possible that pytorch_lightning was left out of the docker image?
I run the container manually and executed python
i was able to import torch, but I was not able to import pytorch_lightning

Little background.
I am running WSL2 ubuntu image on windows. 

docker run --rm -it    -v "$(pwd)"/data:"$(pwd)"/data    -v "$(pwd)"/models:"$(pwd)"/models    --gpus all    ghcr.io/guitarml/pedalnetrt data/ts9_test1_in_FP32.wav data/ts9_test1_out_FP32.wav
Traceback (most recent call last):
  File "train.py", line 1, in <module>
    import pytorch_lightning as pl
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/__init__.py", line 60, in <module>
    from pytorch_lightning import metrics
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/metrics/__init__.py", line 14, in <module>
    from pytorch_lightning.metrics.metric import Metric
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/metrics/metric.py", line 23, in <module>
    from pytorch_lightning.metrics.utils import _flatten, dim_zero_cat, dim_zero_mean, dim_zero_sum
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/metrics/utils.py", line 18, in <module>
    from pytorch_lightning.utilities import rank_zero_warn
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/utilities/__init__.py", line 24, in <module>
    from pytorch_lightning.utilities.apply_func import move_data_to_device
  File "/opt/conda/lib/python3.8/site-packages/pytorch_lightning/utilities/apply_func.py", line 25, in <module>
    from torchtext.data import Batch
ImportError: cannot import name 'Batch' from 'torchtext.data' (/opt/conda/lib/python3.8/site-packages/torchtext/data/__init__.py)

**Comment by mishushakov** [comment]
Interesting, lightning is definitely in the docker image
https://github.com/GuitarML/PedalNetRT/blob/32efd30a0f87d9bf887b7db95a4169ad7afd9db9/Dockerfile#L4

https://github.com/GuitarML/PedalNetRT/blob/32efd30a0f87d9bf887b7db95a4169ad7afd9db9/requirements-docker.txt#L1

are you pulling the image from the repo or building yourself?

**Comment by mishushakov** [comment]
the Dockerfile doesn't specify the pytorch version (1.7.0) so if you're pulling the latest Docker image or building yourself it pulls a newer version, which i suppose isn't compatible

https://github.com/GuitarML/PedalNetRT/blob/32efd30a0f87d9bf887b7db95a4169ad7afd9db9/Dockerfile#L1

didn't occur to me yet, maybe because i have the images cached
it's an easy fix, i'll update the Dockerfile after i verify this is the case

**Comment by creativeguitar** [comment]
I think you might have other things cached, or I am doing something wrong. 
I was pulling the image remotely. I I just downloaded the image and built locally for testing

First problem I had was that I was not able to import pytorch_lightning. This was fixed by installing pytorch_lightning with pip install instead of pip3 install. 

Second problem I had is that the command provided "docker run --rm -it -v "$(pwd)"/data:data -v "$(pwd)"/models:models --gpus all ghcr.io/guitarml/pedalnetrt data/ts9_test1_in_FP32.wav data/ts9_test1_out_FP32.wav"
tries to mount mylocalpath/data into a folder name "data" that doesn't exists inside the container, I got an error saying that I needed to specify an absolute path. I got into the container and created /data and /models folder and then add the "/" in front of the path 

my third problem is not related to this repo. Graphics card is CUDA version 3.0. so when I run python3 train.py. I get an error that my GPU is too old and minimum supported version is 3.5.

I found this repo to build pytorch from source. I will report after testing.
